recipes/paml/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: paml
+  version: "4.9c"
+
+build:
+  number: 0
+
+source:
+  fn: paml-4.9c.tar.gz
+  url: http://abacus.gene.ucl.ac.uk/software/paml4.9c.tgz
+  md5: 852d60dd435bc3b1ce9c47c2a7b83806
+
+requirements:
+  build:
+    - gcc # [linux]
+
+# codeml binary does not accept arguments, so any mock execution produces an
+# exit code !=0. a quick real execution is therefore used as a test case.
```

Comment:
I think so. We do not want to pollute the git repository with example data. If the data is already in your package you could use source-files: http://conda.pydata.org/docs/building/meta-yaml.html#source-files